### ISSUE TITLE
UMD build

### ISSUE THREAD

**Issue by tizmagik** [tizmagik/react-csjs#29]
It would be useful to have a UMD build of `react-csjs` so that samples/playground testing and such can be made in [webpackbin](http://www.webpackbin.com/) for example.

**Comment by wKovacs64** [comment]
Two questions:
1. Do we know why it doesn't work in webpackbin already? A UMD build may come in handy elsewhere, but it doesn't seem like it would be needed for webpackbin since they're just installing and bundling with webpack (other modules work there without UMD builds).
   
   This is the error I got when I tried it:
   
   ```
   ERROR: ModuleNotFoundError
   Module not found: Error: Cannot resolve module 'react-csjs' in /api/sandbox/14772...
   ```
   
   Might just be an issue on their end, too. I see other people have logged similar issues with regards to other modules and at least one of them was closed because the problem went away on its own. Coincidentally, @drcmda has an open issue there with this same error, but his looks unique to nested entry points.
2. Is a UMD build worthwhile without `csjs` having one, too?

## 

That said, I have it working locally. ~~I'll try to open a PR tomorrow.~~

**Comment by tizmagik** [comment]
1. Hm, yea, good point. It's not necessarily specific to Webpackbin. I'm not sure why it doesn't just work out of the box there :/
2. Ah yea, good point. I'm wondering more and more if we should just ship `csjs` with `react-csjs`, it would make a singleton wrapper to support things like #2 easier.

What are your thoughts?

**Comment by wKovacs64** [comment]
Yeah, you might be right. Generally speaking, I'm not a huge fan of shipping an external lib, but in this case, it would certainly make both this and #2 easier. Hmm...

**Comment by tizmagik** [comment]
I would say that it it were just for the UMD build it wouldn't be worth it, but it might make sense to do it once we solve #2
